extensions/replication/utils/QueueEntry.js

```diff
@@ -73,6 +73,20 @@ class QueueEntry extends ObjectMD {
         return this.objectVersionedKey;
     }
 
+    getUserMetadata() {
+        const metaHeaders = {};
+        const data = this.getValue();
+        Object.keys(data).forEach(key => {
+            if (key.startsWith('x-amz-meta-')) {
+                metaHeaders[key] = data[key];
+            }
+        });
+        if (metaHeaders !== {}) {
```

Comment:
This conditional will always be true.

```js
> x = {}
{}
> x === {}
false
> x !== {}
true
```
Best thing is to use `Object.keys.length > 0`

Reply:
Updated to `if (Object.keys(metaHeaders).length > 0)`